### PR TITLE
fix: add description field to CSV base

### PR DESCRIPTION
**Pull request by marioferh**
Add description field to CSV base of every owned CRD to generate correct description when run bundle generate.

### CODE REVIEW

File: bundle/manifests/observability-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -59,40 +59,68 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - kind: AlertmanagerConfig
+    - description: AlertmanagerConfig defines a namespaced AlertmanagerConfig to be
```
Comment (machine424) [inline bundle/manifests/observability-operator.clusterserviceversion.yaml:62]: Did you take a look at the UI rendering? I think it should be shorter to fit into the box, like in Gabriel's example

**Comment by machine424** [comment]
The descriptions are not truncated and the boxes don't look text-heavy.

/lgtm

File: bundle/manifests/observability-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (marioferh) [reply bundle/manifests/observability-operator.clusterserviceversion.yaml:62]: It is ok, thanks!

Comment (sthaha) [reply bundle/manifests/observability-operator.clusterserviceversion.yaml:62]: image

Comment (sthaha) [reply bundle/manifests/observability-operator.clusterserviceversion.yaml:62]: You can use  https://operatorhub.io/preview to preview the bundle/csv.yaml 
I think we can improve the  `AlertmanagerConfig` a bit ..  (ask chatgpt :D )

Comment (sthaha) [reply bundle/manifests/observability-operator.clusterserviceversion.yaml:62]: BTW, are we now supporting all these API?

Comment (marioferh) [reply bundle/manifests/observability-operator.clusterserviceversion.yaml:62]: In a cluster this is the result:
image

Comment (marioferh) [reply bundle/manifests/observability-operator.clusterserviceversion.yaml:62]: `"AlertmanagerConfig is used to define configurations for Prometheus Alertmanager, specifying how alerts are handled and routed for notifications."`

Something like this?

Comment (sthaha) [reply bundle/manifests/observability-operator.clusterserviceversion.yaml:62]: @simonpasquier would be the best judge of this :) .. BTW, for the record, this was only a suggestion and shouldn't block the PR from merging.

Comment (simonpasquier) [reply bundle/manifests/observability-operator.clusterserviceversion.yaml:62]: small suggestion compared to @marioferh 

> ... specifying how alerts are grouped and notified to external systems.

Comment (marioferh) [reply bundle/manifests/observability-operator.clusterserviceversion.yaml:62]: @simonpasquier https://github.com/prometheus-operator/prometheus-operator/pull/5955

**Comment by jan--f** [comment]
Thanks! I guess there is nothing but keeping these up-to-date manually is there?

**Pull request merged**